Request for code review

javitocor opened the pull request:
In this project we have built different kind of forms (HTML, #form_tag and #form_for) in RoR.
Checking the differences between all of them. To do this we have created a users model and controller with new and edit actions and views.
We also have added a partial to display validation errors and as extra credit, the input(s) with errors are highlighted.
All rubocop errors have been fixed.

Redvanisation commented on app/views/users/new.html.erb:
```diff
@@ -0,0 +1,48 @@
+
+
+
+<% if false %>
+  <%= form_tag('/users', method:'post') do %>
+      <%= label_tag 'Email' %>
+      <%= email_field(:user, :email) %><br>
+      <%= label_tag 'Name' %>
+      <%= text_field_tag(:username) %><br>
+      <%= label_tag 'Password'%>
+      <%= password_field_tag(:password) %><br>
+      <%= submit_tag('Submit User') %>
+  <% end %>
+<% end %>
```
This form should be commented out.

Redvanisation commented on app/views/users/new.html.erb:
```diff
@@ -0,0 +1,48 @@
+
+
+
+<% if false %>
+<form accept-charset="UTF-8" action="/users" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="email">
+    Email:
+    <input type="email" name="user[email]">
+  </label>
+  <br>
+  <label for="username">
+    Username:
+    <input type="text" name="user[username]">
+  </label>
+  <br>
+  <label for="password">
+    Password:
+    <input type="password" name="user[password]">
+  </label>
+  <br>
+  <input type="submit" value="Submit">
+</form>
```
This form should be commented out as well.

javitocor commented:
Commented out the html and tag_form in new.html.erb
Removed comment in users_controller
Added extra-features branch with some bootstrap styling (in progress)
